### DISCUSSION TITLE
Help: The ipv6 address obtained from the online server is incomplete.

### DISCUSSION THREAD

**Discussion by Junt62** [qdm12/ddns-updater#435] [General]
When I type "Curl https://api6.ipify.org" in the Ubuntu background.

I got result kind "2408:8256:480:3162::cef"

But ddns-updater's webui told me it detected "2408:8256:480:3162::"

Please ask me for any information you need.

**Comment by Junt62** [comment]
![screenshot-20230216-164027](https://user-images.githubusercontent.com/40317014/219312782-af34aac3-f288-419e-a4ed-caeaeaeb76c6.png)
![screenshot-20230216-164120](https://user-images.githubusercontent.com/40317014/219312789-7af52ce2-a3c0-434c-bcee-ccd0c7fd755c.png)

**Reply by imwaltraud** [reply]
I have the same issue

**Reply by qdm12** [reply]
Fixed in 954dffd3a78751061c01e0f9135086a306f57dc2 for the latest image and future release v2.6.0

**Comment by qdm12** [comment]
Fixed in 954dffd3a78751061c01e0f9135086a306f57dc2 for the latest image and future release v2.6.0
